tests/tools.vitruv.applications.umljava.tests/META-INF/MANIFEST.MF

```diff
@@ -18,6 +18,7 @@ Require-Bundle: org.junit.jupiter.api,
  org.eclipse.xtend.lib,
  org.apache.log4j,
  org.emftext.language.java.resource.java,
- org.emftext.language.java.resource
+ org.emftext.language.java.resource,
+ tools.vitruv.applications.testutility;bundle-version="0.3.0"
```

Comment:
```suggestion
 tools.vitruv.applications.testutility
```

Reply:
Removed in 06b957375b3ca51382fc2f14ed33ec5557c2b89b